html/editing/editing-0/spelling-and-grammar-checking/references/spelling-markers-011-ref.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en" class="reftest-wait">
+<meta charset="utf-8">
+<title>Reference file for spellcheck tests</title>
+<style>
+  div {
+    padding: 10px;
+    background: silver;
+  }
+</style>
+
+<p>The test passes if you see a spelling error marker in the word "quikc".</p>
+<div id="test" contenteditable=true spellcheck=false>The quikc brown fox.</div>
+
+<script>
+  var test = document.getElementById("test");
+  test.addEventListener("focus", () => {
+    document.documentElement.classList.remove("reftest-wait");
```

Comment:
The ref here can be simplified by not bothering with the focus or "reftest-wait" at all. Or is there a cursor that needs to end up in the right place? If so, can the blinking cursor not cause the test to fail, and can it be hidden?